analyzers/rspec/vbnet/S3966.html

```diff
@@ -0,0 +1,15 @@
+<h2>Why is this an issue?</h2>
```

Comment:
This change is irrelevant. There's something wrong with the PR (wrong target branch, missing squash, missing note about what not to review)